modules/ept-md-parser/client/parseFactory.js

```diff
@@ -3,13 +3,18 @@ var renderer;
 
 function parse(input) {
   if (!input) { input = ''; }
-
   // escape double forward slashes before parsing markdown
   input = input.replace(/\/\//g, '&#47;&#47;');
+
+  // manually handle newlines
+  input = input.replace(/\n/g, String.fromCharCode(26));
+
   // compile markdown
   input = compiler(input, { renderer: renderer });
+
```

Comment:
include arbitrary newline changes in a separate commit. review this and either specify that you intended this change, or undo it.

Reply:
These are all intended